### PR TITLE
Release Process FAQ

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/process/release-faq.rst [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+**Q14: How does Redux branching work?**
+
+ - A: https://pre-release.mattermost.com/core/pl/uc31wdd413d58gdxh1jwxidoxc Should be documented in Dev release process
+
+**Q15: How does translations branching work?**
+
+ - A: Lock the translation server to the release branch. The translation PR will be done against the release branch and we can just merge it, forget about master and cherry-picking at this point, then when we lock the translation server back to master the next PR against master will include those translations that went in for the release branch.
+
+**Q16: How does cutting mobile builds work?**
+
```
Comment (amyblais) [inline source/process/release-faq.rst]: There was discussion about this here but the conclusion wasn't posted in the channel - https://pre-release.mattermost.com/core/pl/duw9ifj7siyfbqzgr5deribrto. @enahum Not urgent, when you have a moment can you help write the process here?

File: source/process/release-faq.rst [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+ - A:
+
+**Q13: What changes were made to the team release process to account for the rotating feature and bug fix releases? https://docs.mattermost.com/process/release-process.html**
+
+ - A:
+
+**Q14: How does Redux branching work?**
+
+ - A: https://pre-release.mattermost.com/core/pl/uc31wdd413d58gdxh1jwxidoxc Should be documented in Dev release process
```
Comment (amyblais) [inline source/process/release-faq.rst]: This one might be good to document somewhere in the Dev release process doc if not already. @cpanato

File: source/process/release-faq.rst [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+ - A: Usually features / Story tickets.
+
+**Q11: How does release team monitor what changes went into a release?**
+
+ - A: Monitor the commit history of the respective release branch, e.g. https://github.com/mattermost/mattermost-server/commits/release-5.4 contains commits that shipped with mattermost-server v5.4. Jira ticket is resolved after cherry picking is done.
+ - A: Jira query for tickets that need to wait for next release for testing and to remember to cherry-pick?
+
+**Q12: What changes were made to the dev release process to account for the rotating feature and bug fix releases? https://developers.mattermost.com/internal/release-process/**
+
+ - A:
```
Comment (amyblais) [inline source/process/release-faq.rst]: @cpanato The Dev release process might need some updates - let me know when those are done (they don't all need to be listed here, just some key ones).

Comment (cpanato) [reply source/process/release-faq.rst]: should i open a PR to make this changes in this file?

Comment (amyblais) [reply source/process/release-faq.rst]: Yes, it would make sense if there are any changes being made to the dev release process.

Comment (cpanato) [reply source/process/release-faq.rst]: ok, will review this next week

**Pull request merged**